app/assets/javascripts/application.js

```diff
@@ -14,4 +14,4 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
-//= require bootstrap-sprockets
\ No newline at end of file
+= require bootstrap-sprockets
```

Comment:
I can't find an easy how-to guide which shows this, but you'll see examples dotted around on http://guides.rubyonrails.org/asset_pipeline.html